forge-gui/res/cardsfolder/upcoming/transmogrant_altar.txt

```diff
@@ -0,0 +1,8 @@
+Name:Transmogrant Altar
+ManaCost:3
+Types:Artifact
+A:AB$ Mana | Cost$ B T Sac<1/Creature> | Produced$ C | Amount$ 3 | SpellDescription$ Add {C}{C}{C}.
+A:AB$ Token | Cost$ 2 T Sac<1/Creature> | TokenScript$ c_3_3_a_zombie | SorcerySpeed$ True | SpellDescription$ Create a 3/3 colorless Zombie artifact creature token. Activate only as a sorcery.
+DeckHints:Ability$Token
```

Comment:
is this as fuel for saccing?
you can also add a Sac preference thing for the AI if you want